frontend/src/components/IstioWizards/RequestTimeouts/RouteRetry.tsx

```diff
@@ -67,7 +68,7 @@ export class RouteRetry extends React.Component<RouteRetryProps> {
               />
               <FormHelperText>
                 <HelperText>
-                  <HelperTextItem>{isValid(this.props.isValidRetry) ? tryTimeoutMsg : tryTimeoutMsg}</HelperTextItem>
+                  <HelperTextItem>{isValid(this.props.isValidRetry) ? t(tryTimeoutMsg) : t(tryTimeoutMsg)}</HelperTextItem>
```

Comment:
the same here.
```suggestion
                  <HelperTextItem>{t(tryTimeoutMsg)}</HelperTextItem>
```

Reply:
As you suggested, the code here should indeed be modified. I have completed the modification, please review it when you have time